spec/factories/users.rb

```diff
@@ -2,9 +2,9 @@
 
 FactoryBot.define do
   factory :user do
-    first_name { Faker::Name.first_name }
-    last_name { Faker::Name.last_name }
-    username { Faker::Internet.unique.username }
+    first_name { Faker::Name.first_name[0..49] }
+    last_name { Faker::Name.last_name[0..49] }
+    username { Faker::Internet.unique.username[0..14] }
```

Comment:
`username` takes a `specifier` named parameter which does this:
```ruby
username { Faker::Internet.unique.username(specifier: 5...10) }
```
will give you something in the 5-10 character range.